chukwuemekachm/GraphQL-API
# 24 Implements create author resolvers

#### What does this PR do?
- This PR implements the `createAuthor` mutation.
#### Description of Task to be completed?
- Add createAuthor resolver
- Validate user inputs
- Check for existence of a user with similar details
- Create author and return details of newly created authors
#### How should this be manually tested?
- Set up the repo locally.
- Create a `createAuthor` mutation
- Observe the result and behaviours.
#### Any background context you want to provide?
#### What are the relevant Github issues?
- [#24](https://github.com/chukwuemekachm/GraphQL-API/issues/24)
#### Screenshots (if appropriate)


<img width="1440" alt="screenshot 2019-01-25 at 2 15 49 pm" src="https://user-images.githubusercontent.com/33798252/51779200-2cffe900-2106-11e9-9e77-159e0ebc5bf4.png">


<img width="1440" alt="screenshot 2019-01-25 at 2 15 39 pm" src="https://user-images.githubusercontent.com/33798252/51779201-2cffe900-2106-11e9-86c7-7dea1403166b.png">

<img width="1440" alt="screenshot 2019-01-25 at 2 15 23 pm" src="https://user-images.githubusercontent.com/33798252/51779239-6e909400-2106-11e9-80b6-06e4bdddab19.png">


#### Questions:
- N/A
